inscriptions/ISic003613.xml

```diff
@@ -187,11 +192,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
+                <ab>
+                    <lb n="1"/>Στ<unclear>ρ</unclear>ά<unclear>τ</unclear>
+                    <lb n="2" break="no"/>ο̄νο<unclear>ς</unclear> ἐ
+                    <lb n="3" break="no"/>μι <unclear>σ</unclear>ε͂
+                    <lb n="4" break="no"/>μα <orig>ΤΥ</orig><gap reason="lost" quantity="1" unit="character"/>
+                </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app>
+                        <note>Text after Chaniotis, Corsten, Stroud, and Tybout 2008 (SEG 58.1042)</note>
```

Comment:
although note that they say: 'we give the text of Dubois in BE 2009: 567 (slightly corrected from the ed pr)'. There is a general principle with SEG, which is that unless they explicitly say they correct, they only report texts, and do not wish to be credited with the reading as a result...